test/runtests.jl

```diff
@@ -160,18 +160,102 @@ else  # run HiGHS tests
             @test r["ElectricStorage"]["size_kwh"] ≈ 83.3 atol=0.1
         end
 
-        # TODO test MPC with outages
```

Comment:
@toddleif did you remove this test (or change it to the condensed version below), or is this a product of some merge with develop?

Reply:
That appears to be a product of a merge with develop -- I'm not sure why it was deleted. I added it back in to the code.